kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -31,6 +32,12 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
 
   def this() = this(classOf[KyuubiSessionManager].getSimpleName)
 
+  override def LOG_ROOT: String = if (Utils.isTesting) {
```

Comment:
we don't need this anymore, we can set SERVER_OPERATION_LOG_DIR_ROOT in test or maven pom file